arch/arm/soc/nxp_lpc/lpc54xxx/soc.c

```diff
@@ -35,6 +35,8 @@
 
 static ALWAYS_INLINE void clkInit(void)
 {
+
+#ifdef CONFIG_SOC_LPC54114
```

Comment:
Wonder if this would be a bit more clear as #ifndef CONFIG_SOC_LPC54114_M0, or do we have a CONFIG_SOC_LPC54114_M4 define?

Maybe a comment, about doing this only on the master/m4

Reply:
Do we want to change the name of the original soc to append `_M4` so it's explicit?

Reply:
I think renaming to _M4 would be good.

Reply:
Done rename to _M4.

Note that SOCs are _M4 and _M0 now, while board has no suffix for M4 configuration and board with M0 configuration has suffix for M0. This is because the name of the board already existed without _M4 suffix before.